deltachat-ios/MessageKit/Layout/TextMediaMessageSizeCalculator.swift

```diff
@@ -27,6 +27,12 @@ import UIKit
 
 open class TextMediaMessageSizeCalculator: MessageSizeCalculator {
 
+    var maxMediaItemHeight: CGFloat {
+        return UIScreen.main.bounds.size.height * 0.8
```

Comment:
On smaller devices in landscape the images still are larger than the screen. Why not use something like this:
Screenheight - NavbarHeight (~44) - InputViewHeight (~100). 
This will never accurate (because inputView does respect safeArea), but this way it is guaranteed  to be smaller than the chat window.

Reply:
yes we can do that. Still it can be that the image is larger than the screenHeight if it has a bad ratio (say 1:8) and text is below. But yor suggestion will fix other cases, +1

Reply:
The problem is that the navigationbar height differs between landscape and portatrait, leading to a unnecessarily small image in landscape if I use a constant subtracted from the screen height. I also wonder if the navigation bar height differs between devices, e.g. iPads and iPhones. I would rather reduce the multiplier a little bit so that the media height keeps being fully relative to the screen size.

Reply:
alright.